packages/side-nav/test/side-nav-item.test.js

```diff
@@ -94,6 +94,41 @@ describe('side-nav-item', () => {
     });
   });
 
+  describe('path aliases', () => {
+    beforeEach(async () => {
+      item = fixtureSync(`<vaadin-side-nav-item path="/path" path-aliases="/"></vaadin-side-nav-item>`);
+      await nextRender();
+    });
+
+    it('should set active property to true for matching alias with path', () => {
+      expect(item.active).to.be.true;
+    });
+
+    it('should set active property to false for matching alias without path', async () => {
+      item.path = undefined;
+      await item.updateComplete;
+      expect(item.active).to.be.false;
+    });
+
+    it('should update active property when aliases are updated', async () => {
```

Comment:
How about we improve the structure?

```js
describe('matching path', () => {
  beforeEach(() => item.path = '/');

  it('should be active');

  it('should be active when no aliases match', () => {
    item.pathAliases = '/alias';
  });
});

describe('not matching path', () => {
  it('should be inactive');

  it('should be active when an alias matches', () => {
    item.pathAliases = '/, /alias';
    expect(item.active);
    item.pathAliases = '/alias, /';
    expect(item.active);
  });

  it('should be active when an empty alias matches');
});

describe('matching path is set initially', () => {

})

describe('matching path alias is set initially', () => {

});
```

Reply:
Updated the structure to be similar to this one. I used 3 categories: 
- no path set initially
- matching path set initially
- non-matching path set initially
I don't have any strong opinions on this, what do you think?

Reply:
Looks good to me 👍